templates/CRM/Contact/Form/Task/SMSCommon.tpl

```diff
@@ -55,25 +60,27 @@ cj('#sms_text_message').bind({
 
 function maxLengthMessage()
 {
-   var len = cj('#sms_text_message').val().length;
+   var len = CRM.$('#sms_text_message').val().length;
    var maxLength = {/literal}{$max_sms_length}{literal};
    if (len > maxLength) {
-      cj('#sms_text_message').crmError({/literal}'{ts escape="js"}SMS body exceeding limit of 160 characters{/ts}'{literal});
+      CRM.$('#sms_text_message').crmError({/literal}'{ts escape="js"}SMS body exceeding limit of {$max_sms_length} characters{/ts}'{literal});
```

Comment:
This might not work with the string extractor (placeholder).

I recommend something like:

```
CRM.$('#sms_text_message').crmError({/literal}'{ts escape="js" 1=>{$max_sms_length}}SMS body exceeding limit of %1 characters{/ts}'{literal});
```

(not tested)

Reply:
Have added https://github.com/civicrm/civicrm-core/pull/20722